src/main/java/com/stuypulse/robot/commands/climber/ClimberMoveCommand.java

```diff
@@ -48,11 +48,10 @@ public void execute() {
     @Override
     public void end(boolean interrupted) {
         climber.setMotorStop();
-        climber.setClimberLocked();
     }
 
     @Override
     public boolean isFinished() {
-        return movingUp ? climber.getTopReached() : climber.getBottomReached();
+        return Settings.Climber.ENABLE_ENCODERS && climber.getTopHeightLimitReached();
```

Comment:
"ENABLE_ENCODERS" should change the behavior of the subsystem, not the commands. 

this just means getTopReached should change its behavior depending of ENABLE_ENCODERS. 

(however do look at my other comment about enable encoders)

Reply:
also check bottom limit here I think